static/src/javascripts/projects/commercial/modules/brazeBanner.js

```diff
@@ -154,14 +155,25 @@ const getMessageFromBraze = async (apiKey: string, brazeUuid: string): Promise<b
             return;
         }
 
-        appboy.subscribeToInAppMessage((message: InAppMessage) => {
+        let subscriptionId: ?string;
+
+        const callback = (message: InAppMessage) => {
             if (message.extras) {
                 messageConfig = message;
                 resolve(true);
             } else {
                 resolve(false);
             }
-        });
+
+            // Unsubscribe
```

Comment:
It's only jumped out to me upon re-reading but this comment is either not needed (in which case we should remove it) or it is needed, in which case it should be more informative then this! Comments are most helpful when they describe intent or context that isn't clear from the source code itself. This one is walking near the line of repeating what the code says.

Super minor point and even if you agree, certainly not something to hold up releasing this PR.

Reply:
You do have a point on that, i'll take it out for now - there's no time like the present.